src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -117,12 +117,44 @@ export class BaseEditCourseForm extends React.Component {
           <Collapsible
             title={this.formatCourseTitle(title, courseStatuses)}
             key="Test Key"
+            isOpen
```

Comment:
I just threw this unrelated change in here - it came out of bug bush.

Reply:
Looks like "Test Key" was originally put in by Jeff and I believe it has to do with it being in a Collapsible and requiring a unique key value. Not sure why it's that and could probably be switched to a uuid or something, but @jlajoie or @bbaker6225  can probably speak better to it.

Reply:
Yeah that shouldn't be this, just make it something unique, sorry for the mistake!

Reply:
You shouldn't need `key` at all since it's not in an iterator